src/Autocomplete/Autocomplete.stories.tsx

```diff
@@ -112,10 +113,10 @@ export const KeyValueOptions: StoryObj<typeof Autocomplete> = {
     margin: "normal",
     multiple: false,
     options: [
-      { key: 1, value: "Option 1" },
-      { key: 2, value: "Option 2" },
-      { key: 3, value: "Option 3" },
-      { key: 4, value: "Option 4" }
+      { key: 1, value: "Option1" },
+      { key: 2, value: "Option2" },
+      { key: 3, value: "Option3" },
+      { key: 4, value: "Option4" }
```

Comment:
We've lost the space in the text here. Any reason?

Reply:
Copy pasted the wrong object.